src/FieldWarning/Assets/Ingame/CaptureZone/CaptureZone.cs

```diff
@@ -0,0 +1,115 @@
+﻿using PFW.Model.Game;
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+namespace PFW.Ingame.UI
+{
+    public class CaptureZone : MonoBehaviour
+    {
+        public Material red;
+        public Material blue;
+        public Material neutral;
+        //How many Points per tick this Zone gives
+        public int Worth = 3;
+        //Maybe take out all that owner stuff and simply use an int or otherwise shorten the code
+        private PlayerData owner;
+        //Vehicles Currently in the Zone (Maybe exclude all non-commander vehicles),maybe replace list with Array
+        private List<VehicleBehaviour> _vehicles = new List<VehicleBehaviour>();
+        // Start is called before the first frame update
+        void Start()
+        {
+        }
+
+        // Update is called once per frame
+        void Update()
+        {
+            //Check if Blue Red None or Both Occupy the Zone
+            bool redIncluded = false;
+            bool blueIncluded = false;
+            PlayerData newOwner = null;
+            for (int i = 0; i<_vehicles.Count;i++)
+            {
```

Comment:
Its unusual but with the current style we newline braces for method/class declarations but write them inline for control flow statements.